crates/teloxide-core/src/types/non_telegram_types/currency.rs

```diff
@@ -348,6 +348,8 @@ pub enum Currency {
     XPT,
     /// SUCRE
     XSU,
+    /// Telegram Stars
+    XTR,
```

Comment:
Please, add conversion to the string for `Currency` too (if there is some magic-crate that converts unit variants to strings without a lot of hand written things)